review-tools/gitaddrev

```diff
@@ -150,6 +150,11 @@ foreach (@ARGV) {
         $WHAT = 'perftools';
         $min_authors = 2;
         $min_otc = 0;
+    } elsif (/--installer$/) {
+        $WHAT = 'installer';
+        $min_authors = 1;
+        # openssl/installer is governed by OTC
+        $min_otc = 1;
         $min_omc = 0;
```

Comment:
These are the defaults. You removed `min_omc` from `--perftools` and left `min_otc` which is strange (but it doesn't really matter since they are the default)

Reply:
made this match the other repos